server/server.js

```diff
@@ -23,6 +24,7 @@ const s3Headers = {
 const pathname = path.join(__dirname, '..', 'public');
 app.use(express.static(pathname));
 app.use(cors());
+app.use(bodyParser.json());
```

Comment:
I'm kind of surprised we hadn't run into issues yet without bodyParser.

Reply:
Hahaha, we haven't had to post anything yet!